src/platforms/java/common/configuration/transport.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Transport
+sidebar_order: 11
+description: "Learn how to create and configure alternative HTTP transport implementation"
+---
+
+Sentry Java SDK by default comes with a lightweight transport implementation based on [HttpUrlConnection](https://docs.oracle.com/javase/8/docs/api/java/net/HttpURLConnection.html). While this implementation is a great fit for the error reporting feature, it may provide enough throughput for high traffic, server applications using the Performance feature.
```

Comment:
```suggestion
Sentry Java SDK by default comes with a lightweight transport implementation based on [HttpUrlConnection](https://docs.oracle.com/javase/8/docs/api/java/net/HttpURLConnection.html). While this implementation is a great fit for the error reporting feature, it may provide enough throughput for high traffic, server applications using Sentry's Performance Monitoring.
```